packages/shared/styles/components/atoms/SfButton.scss

```
@@ -4,104 +4,152 @@
-  transition: var(--button-transition, background 150ms linear);
-  cursor: pointer;
+  transition: var(--button-transition, box-shadow 150ms ease-in-out);
+  cursor: var(--button-cursor, pointer);
   outline: none;
-  &:hover {
-    --button-background: var(--c-primary-variant);
-  }
+
+  @include font(
+    --button-font,
+    var(--font-weight-bold),
+    var(--font-size-regular),
+    1.6,
+    var(--body-font-family-secondary)
+  );
+
+  @include border(--button-border, 0, solid, var(--c-primary));
+
   @each $color, $map in $colors-map {
     $primary: map-get($map, "primary");
     $variant: map-get($map, "variant");
     &.color-#{$color} {
       --button-background: #{$primary};
-      &:hover {
-        --button-background: var(--c-#{$color}-lighten);
+      &:active {
+        --button-background: #{$variant};
       }
     }
   }
+
+  &:hover {
+    --button-box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  }
+
+  &:active {
+    --button-box-shadow: none;
+    --button-background: var(--c-primary-variant);
+    --button-transition: background 150ms linear;
+  }
+
   &:disabled {
-    --button-background: var(--c-gray);
-    cursor: default;
-    &:hover {
-      --button-background: var(--c-gray);
-    }
+    --button-background: var(--c-light);
+    --button-color: var(--c-text-muted);
+    --button-cursor: default;
+    --button-box-shadow: none;
   }
+
   &--full-width {
     --button-width: 100%;
   }
+
   &--outline {
-    --button-padding: calc(var(--spacer-medium) - 1px)
-      calc(var(--spacer-extra-big) - 1px);
-    --button-border: 1px solid var(--c-primary);
+    --button-border-width: 2px 2px 2px 2px;
     --button-background: transparent;
-    --button-color: var(--c-primary);
-    &:hover {
-      --button-background: var(--c-primary);
-      --button-color: inherit;
+    --button-color: var(--c-text);
+
+    &:active {
+      --button-border-color: var(--c-primary-variant);
+      --button-background: transparent;
+      --button-color: var(--c-text-muted);
+      --button-transition: color 150ms linear;
+    }
+    &:disabled {
+      --button-border-color: var(--c-light);
+      --button-background: transparent;
+      --button-color: var(--c-text-muted);
     }
     @each $color, $map in $colors-map {
       $primary: map-get($map, "primary");
       &.color-#{$color} {
-        --button-border: 1px solid #{$primary};
+        --button-border-color: #{$primary};
         --button-background: transparent;
-        --button-color: #{$primary};
-        &:hover {
-          --button-background: #{$primary};
-          --button-color: inherit;
+        &:active {
+          --button-background: transparent;
         }
       }
     }
-    &:disabled {
-      --button-border: 1px solid var(--c-gray);
-      --button-background: transparent;
-      --button-color: var(--c-gray);
-    }
   }
+
   &--text {
-    --button-border: 0;
+    --button-border-width: 0;
     --button-padding: var(--spacer-small) 0;
```

```suggestion
    --button-padding: 0;
```
@aniamusial for `--text` we can set padding: 0